lib/rules/global-require.js

```diff
@@ -52,9 +52,25 @@ function isShadowed(scope, node) {
 /** @type {import('../shared/types').Rule} */
 module.exports = {
     meta: {
-        deprecated: true,
-
-        replacedBy: [],
+        deprecated: {
+            message: "Node rule were moved out of ESLint core.",
+            url: "https://eslint.org/docs/latest/use/migrating-to-7.0.0#deprecate-node-rules",
+            deprecatedSince: "7.0.0",
+            availableUntil: null,
+            replacedBy: [
+                {
+                    message: "ESLint Stylistic Maintains now deprecated node rules",
```

Comment:
```suggestion
                    message: "ESLint Stylistic maintains now deprecated node rules",
```

In other places as well.